services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+import init_django_orm  # noqa: F401
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
```

Comment:
Actually, this function also features a mistake in return type annotation.

Reply:
Agree. Changed to object.

Reply:
Thanks for the quality review.!

Reply:
We'll see if it was a quality review after mentor's review. :)

Reply:
100%